.github/workflows/compiler_macos_build_and_test.yml

```diff
@@ -89,6 +89,10 @@ jobs:
       - name: Test
         run: |
           set -e
+          export KEY_CACHE_DIRECTORY=$(mktemp -d)/KeySetCache
+          echo "KEY_CACHE_DIRECTORY=$KEY_CACHE_DIRECTORY" >> "${GITHUB_ENV}"
+          mkdir $KEY_CACHE_DIRECTORY
+
```

Comment:
Shouldn't we add it to the non-macos tests as well ?

Reply:
non-mac-os test are ondemand instance